pyproject.toml

```diff
@@ -19,9 +19,43 @@ repository = "https://github.com/gis-ops/py-osrm"
 [tool.scikit-build]
 # Protect the configuration against future changes in scikit-build-core
 minimum-version = "0.4"
-
 # Setuptools-style build caching in a local directory
 build-dir = "build/{wheel_tag}"
-
 # Build stable ABI wheels for CPython 3.12+
 wheel.py-api = "cp312"
+
+[tool.cibuildwheel]
+test-command = "pytest"
+skip = "*musllinux*"
+archs = "native"
+manylinux-x86_64-image = "ghcr.io/gis-ops/manylinux_2_28_x86_64_py-osrm"
+
+[tool.cibuildwheel.linux]
+build = "cp38-*"
+test-requires = "pytest"
+test-command = "pytest {project}/tests"
+repair-wheel-command = "LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib/intel64/gcc4.8/ auditwheel repair -w {dest_dir} {wheel}"
+before-test = "osrm-datastore ./tests/test_data/ch/monaco"
+before-all = """
+    # Install TBB &&
+        wget --tries 5 https://github.com/oneapi-src/oneTBB/releases/download/v2021.3.0/oneapi-tbb-2021.3.0-lin.tgz -O onetbb.tgz &&
+        tar zxvf onetbb.tgz &&
+        cp -a oneapi-tbb-2021.3.0/lib/. /usr/local/lib/ &&
+        cp -a oneapi-tbb-2021.3.0/include/. /usr/local/include/ &&
+        export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib/intel64/gcc4.8/ &&
+    # Install Lua 5.2 &&
```

Comment:
what was the issue with lua again? on the manylinux image with almalinux 8 we should have 5.3.4. available, is that not compatible with osrm?

Reply:
For some reason, the default `lua-devel` from dnf seems to cause an error with sol, on [this line](https://github.com/Project-OSRM/osrm-backend/blob/3bb82ce1e2fd2299712b96176e651e6f7e999aae/third_party/sol2-3.3.0/include/sol/sol.hpp#L3110) if I recall correctly.